src/c++/perf_analyzer/docs/examples/decoupled_output_file.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
I feel like having just one example would be fine. Doesn't seem useful to user to see what a misformatted output looks like. Also doesn't seem useful to show both a coupled and decoupled. I think they'll understand that the responses can be one or multiple because it's an array and they could also look at the schema file.

Reply:
removed known_error_file.json, coupled_output_file.json